### PR TITLE
Add view with Gitpod Flex instructions

### PR DESCRIPTION
**Pull request by jeanp413**
## Description

Add view with Gitpod Flex instructions

## Related Issue(s)

Fixes https://linear.app/gitpod/issue/NEXT-2305/[vscode]-add-new-webview-in-classic-with-gitpod-flex-instructions

## Documentation

/hold

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -379,6 +379,11 @@
 					"name": "Workspace",
 					"icon": "$(squirrel)",
 					"when": "false"
+				},
+				{
+					"id": "gitpod-flex-help",
+					"name": "Help",
```
Comment (akosyakov) [inline package.json:385]: Maybe better something like `Looking for Gitpod Flex?`

File: package.json [hunk 2]
```diff
@@ -387,6 +392,10 @@
 				"view": "gitpod-login",
 				"when": "gitpod.authenticated != true && !gitpod.inGitpodFlexRemoteWindow",
 				"contents": "You have not yet signed in with Gitpod\n[Sign in](command:gitpod.signIn)"
+			},
+			{
+				"view": "gitpod-flex-help",
+				"contents": "Connecting to Gitpod Flex environments requires installing the [Gitpod Flex extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-flex) and opening them through the [Gitpod Flex](https://app.gitpod.io/). This view is only to manage [Gitpod Classic](https://gitpod.io/workspaces) workspaces."
```
Comment (akosyakov) [inline package.json:398]: `This view is only to manage [Gitpod Classic](https://gitpod.io/workspaces) workspaces.` can we visually separate this sentence, i.e. like a new paragraph.

**Comment by akosyakov** [comment]
@jeanp413 important point that it should be only for gitpod.io users. Enterprise users should not see it.

**Comment by jeanp413** [comment]
image

**Review by akosyakov — APPROVED** [review]

**Pull request merged**